key/aziot-key-client-async/src/lib.rs

```diff
@@ -95,7 +95,7 @@ impl Client {
 
         http_common::request_no_content(
             &self.inner,
-            http::Method::GET,
+            http::Method::DELETE,
```

Comment:
Oops.

`delete_key` is wrong too, so fix that too please.

Reply:
Fixed